occurrences/tests/test_api.py

```diff
@@ -686,7 +712,20 @@ def test_delete_venue_staff_user(staff_api_client, venue):
       groupSize
       groupName
       amountOfAdult
-      studyLevel
+      studyLevels {
+        edges {
+            node {
+                id
+                originalId
```

Comment:
Is this originalId a good solution? In my opinion there is no need for GraphQL id at all here, because StudyLevel id is an assignable string.

Reply:
Good point, I think we can just get rid of the graphQL global ID, then use `id` to display this originalId

Reply:
I found out that solution like this `variables={"id": to_global_id("StudyGroupNode", study_group.id)}` is used to create global id in tests. Didn't know that earlier. Should we use the global graphql id instead of database id? Or should we support them both?

Reply:
oh, sorry @quyenlq. My view didn't show your response 👍

Reply:
Yeah the goal of global ID comes from Relay, where encoded ID includes the Node type so Client-side can confirm its type immediately without further investigation. If we don't want this behavior, then we can just remove the `relay.Node` interface in the `StudyLevelNode`. One more note, we have a convention that only Types that implement `relay.Node` interface will have the suffix *Node, so consider to change the class name too :)